package.json

```diff
@@ -1,10 +1,10 @@
 {
   "private": true,
   "scripts": {
-    "version-stamp": "lerna run version-stamp",
-    "build": "lerna run build --no-private",
```

Comment:
Why are you removing `--no-private` here?

Reply:
Don't we want to build everything to be safe?
This was reverted in another branch btw.

Reply:
Reverted in this branch as well.